Makefile-docker

```diff
@@ -156,13 +158,13 @@ setup-ui-tests:
 	# Create a proper superuser that can be used to access the API
 	python manage.py waffle_switch super-create-accounts on --create
 	python manage.py waffle_switch activate-autograph-signing on --create
-	python manage.py createsuperuser --email=uitest@restmail.net --username=uitest --noinput --add-to-supercreate-group --hostname=olympia-frontend.test
 
 ui-tests: setup-ui-tests
```

Comment:
As shown in my recent change in your branch, let's name this `run-ui-tests` to make this action more explicit.